src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/IUIImageService.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree;
+
+internal interface IUIImageService : IVsUIService<SVsImageService, IVsImageService2>
```

Comment:
@tmeschter " isolate the VS dependencies behind a new interface"

do we want to just put this behind a new interface 
or 
do we want to create a new mef exported service to complete isolate the vs service and only expose specific methods ?